lib/src/layer.dart

```diff
@@ -97,12 +146,14 @@ class Layer {
       px = 0;
       
       cols.asMap().forEach((j, Tile t) {
-        tileId = tileMatrix[j][i];
+        tileId = tileMatrix[i][j];
+        rotation = tileRotation[i][j];
         tile = map.getTileByGID(tileId)
-        ..x = i
-        ..y = j
+        ..x = j
+        ..y = i
```

Comment:
why are you changing i/j x/y on this PR? is this related?

Reply:
no sry, in my testing the map was rotated by 90° 
I think it just was a flaw in my testing I'll revert that